packages/gatsby-remark-copy-linked-files/src/index.js

```diff
@@ -44,9 +44,12 @@ module.exports = ({ files, markdownNode, markdownAST, getNode }) => {
     visitor(link)
   })
 
-  // Also copy gifs since Sharp can't process them.
+  // Also copy gifs since Sharp can't process them,
+  // and svgs since we exclude them from the image processing pipeline in
+  // gatsby-remark-responsive-image
   visit(markdownAST, `image`, image => {
-    if (image.url.slice(-3) === `gif`) {
+    const fileType = image.url.slice(-3)
```

Comment:
This is unnecessarily fragile code now that I think about it... we can just get the actual extension off `linkNode` as we do above.